cli/cmd/test_connection.go

```diff
@@ -22,6 +30,52 @@ cloudquery test-connection ./directory ./aws.yml ./pg.yml
 `
 )
 
+func getSyncTestConnectionAPIClient() (*cloudquery_api.ClientWithResponses, error) {
+	authClient := apiAuth.NewTokenClient()
+	if authClient.GetTokenType() != apiAuth.SyncTestConnectionAPIKey {
+		return nil, nil
+	}
+
+	token, err := authClient.GetToken()
+	if err != nil {
+		return nil, err
+	}
+	return api.NewClient(token.Value)
+}
+
+func updateSyncTestConnectionStatus(ctx context.Context, logger zerolog.Logger, status cloudquery_api.SyncTestConnectionStatus) {
+	apiClient, err := getSyncTestConnectionAPIClient()
+	if err != nil {
+		logger.Warn().Err(err).Msg("Failed to get sync test connection API client")
+		return
+	}
+	if apiClient != nil {
```

Comment:
```suggestion
	if apiClient == nil {
		return
	}
```

To unindent the rest

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/17212/commits/7d55b90f80b00b2886c79ddd81ecd5ade09fa82e